documentation/book/con-accessing-kafka-bridge-from-outside.adoc

```diff
@@ -0,0 +1,28 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-using-the-kafka-bridge.adoc
+
+[id='con-accessing-kafka-bridge-from-outside-{context}']
+
+= Accessing the {ProductName} Kafka Bridge from outside of {ProductPlatformName}
+
+After deployment, the {ProductName} Kafka Bridge can only be accessed by applications running in the same {ProductPlatformName} cluster. These applications use the `_kafka-bridge-name_-bridge-service` service to access the API.
+
+If you want to make the Kafka Bridge accessible to applications running outside of the {ProductPlatformName} cluster, you can expose it manually by using one of the following features:
+
+* {KubernetesName} Services of types LoadBalancer or NodePort
+
+* {KubernetesName} Ingress resources
+
+* {OpenShiftName} Routes
+
+If you decide to create Services, use the following labels in the `selector` to configure the pods to which the service will route the traffic:
+
+[source,yaml,subs=attributes+]
+----
+  # ...
+  selector:
+    strimzi.io/cluster: kafka-bridge-name
```

Comment:
The `kafka-bridge-name` has to be replaced by the actual name of the CR. Do we will mark the things which should be replaced as `<kafka-bridge-name>`? Or just mark it with this `<1>` and explain below it how it should be replaced like you do on other places.

Reply:
I think we're a bit inconsistent in our approach. I like using annotations in the YAML examples.  I'll add a `<1>` annotation and an explanation.